Common/src/main/java/com/tabnineCommon/chat/commandHandlers/chatState/ChatState.kt

```diff
@@ -5,7 +5,15 @@ import com.intellij.ide.util.PropertiesComponent
 
 private const val CHAT_CONVERSATIONS_KEY = "com.tabnine.chat.conversations.v2"
 
-data class ChatMessageProps(val id: String, val text: Any, val isBot: Boolean, val timestamp: String, val selected: Int?)
+data class ChatMessageProps(
+    val id: String,
+    val text: Any,
+    val isBot: Boolean,
+    val timestamp: String,
+    val selected: Int?,
+    val intent: Any?
```

Comment:
this is backward compatible, right?

Reply:
i added serialization tests that verify precisely these cases of backwards compatibility in serialization - we're working with the same serializer in the entire plugin.
see my [next commit here](https://github.com/codota/tabnine-intellij/pull/607/commits/3bd98426b962c0665de8b626605c736a0d9d124c)